### PR TITLE
Resolves Spelling / Grammar on the menu interactions page

### PR DESCRIPTION
**Pull request by jordynmackool**
Fixes issue #62

### CODE REVIEW

File: docs/Best Practices/Menu Interactions/index.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ PerformInteraction lists should always be as small as possible by UX design. How
 	
 	- For each item which has several more actions available:  
   Example: Searching for events in the vicinity. There might be a big number of events. On each event you can select “Call venue”, “Navigate to it”, “Details”, “Play” (playing the artist of a potential concert in the vicinity).
-		- Present each result on it's own screen via `Show` and announce it with a pure voice alert via`Speak`.  
+		- Present each result on on its own screen via `Show` and announce it with a pure voice alert via`Speak`.
```
Comment (theresalech) [inline docs/Best Practices/Menu Interactions/index.md:45]:
```suggestion
		- Present each result on its own screen via `Show` and announce it with a pure voice alert via `Speak`.  
```

File: docs/Best Practices/Menu Interactions/index.md [hunk 2]
```diff
@@ -42,13 +42,13 @@ PerformInteraction lists should always be as small as possible by UX design. How
...
   Example: Searching for events in the vicinity. There might be a big number of events. On each event you can select “Call venue”, “Navigate to it”, “Details”, “Play” (playing the artist of a potential concert in the vicinity).
-		- Present each result on it's own screen via `Show` and announce it with a pure voice alert via`Speak`.  
+		- Present each result on on its own screen via `Show` and announce it with a pure voice alert via`Speak`.  
        - Define softbuttons for each possible action and add the respective command to voice and menu.
        - To cycle through the result list, you may use one of the below options:
        - `Skip`button (which is currently only available for Media Apps)
        - Softbuttons & Voice commands
        - Show the current item and the length of the list in the Media-track. For example 1/10 for the first item out of 10 items from the result.
-       - Announce the availablility of `Skip`hard button on the steering wheel for easy navigation.
+       - Announce the availability of `Skip`hard button on the steering wheel for easy navigation.
```
Comment (theresalech) [inline docs/Best Practices/Menu Interactions/index.md:51]:
```suggestion
       - Announce the availability of `Skip` hard button on the steering wheel for easy navigation.
```

**Review by theresalech — CHANGES_REQUESTED** [review]

File: docs/Best Practices/Menu Interactions/index.md [hunk 3]
```diff
@@ -42,13 +42,13 @@ PerformInteraction lists should always be as small as possible by UX design. How
 	
 	- For each item which has several more actions available:  
   Example: Searching for events in the vicinity. There might be a big number of events. On each event you can select “Call venue”, “Navigate to it”, “Details”, “Play” (playing the artist of a potential concert in the vicinity).
-		- Present each result on it's own screen via `Show` and announce it with a pure voice alert via`Speak`.  
+		- Present each result on its own screen via `Show` and announce it with a pure voice alert via `Speak`.  
        - Define softbuttons for each possible action and add the respective command to voice and menu.
        - To cycle through the result list, you may use one of the below options:
        - `Skip`button (which is currently only available for Media Apps)
        - Softbuttons & Voice commands
```
Comment (theresalech) [inline docs/Best Practices/Menu Interactions/index.md:49]:
```suggestion
       - Softbuttons & voice commands
```

**Review by theresalech — CHANGES_REQUESTED** [review]
I found a few more minor typos that can be resolved in this PR.

**Comment by jordynmackool** [comment]
Committed changes based on review suggestions. Ready for re-review @theresalech.

**Review by theresalech — APPROVED** [review]

**Pull request merged**